src/vm/wren_core.wren

```diff
@@ -302,6 +302,22 @@ class List is Sequence {
     }
     return result
   }
+
+  sort() { sortStep_(this) }
+
+  sortStep_(l) {
+    if (l.isEmpty) return l
+    return sortStep_(l[1..-1].where {|x| x < l[0] }.toList) + [l[0]] + 
+           sortStep_(l[1..-1].where {|x| x >= l[0] }.toList)
```

Comment:
We're going to trigger allocations for each step in the sort this way. Would it be possible to do an in-place sort?